superset-frontend/src/views/datasetList/Modal.tsx

```diff
@@ -0,0 +1,124 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import styled from '@superset-ui/style';
+import { Modal as BaseModal } from 'react-bootstrap';
+import { t } from '@superset-ui/translation';
+import Button from 'src/components/Button';
+
+interface ModalProps {
+  children: React.ReactNode;
+  disableSave: boolean;
+  onHide: () => void;
+  onSave: () => void;
+  show: boolean;
+  title: React.ReactNode;
+}
+
+const StyledButton = styled(Button)`
```

Comment:
That doesn't seem right. It seems like the style for a button should not be defined in the Modal that contains it. Can't we just use a `<Button/>`?